modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/OpenAPIV3Parser.java

```diff
@@ -251,6 +256,8 @@ private SwaggerParseResult resolve(SwaggerParseResult result, List<Authorization
                     }
                 }
             }
+        }  catch (HostDeniedException error) {
```

Comment:
Do we need to rethrow this in this case or is it okay to just let is be added to the result messages in the `catch (Exception e)` block beneath?